Cleanup

Oxygemo opened the pull request:
This PR focuses on making things cleaner. I am still working on it, some stuff is (unfortunately) broken. Fixing broken things is fun though!

Commit:
Rebrand Deaderine site to Debia

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Commit:
Put footer/header into a src folder and favicon

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Commit:
Redirect dev pages to the new TTS ones and fix the favicon

Redirecting to the TTS pages should give us less stress having to update both pages. We clearly haven't bothered doing that.

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Commit:
Set up even more notifiers that we're not having bios here anymore

And also tell the user that we've put these in TTS' site now

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Commit:
remove dev bio template

Clean up what's not used!

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Oxygemo commented:
!!!! NOTE !!!!: I don't plan on solving #8 with this PR. I'm just cleaning up the files to make sure everything's sorted and not bloated and just overall better.

Commit:
add redirect to the new tts bio collection page

User can click this button and be sent to the new TTS bio page instead.

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Oxygemo commented:
I'm done now! @Rexogamer @Devnol I'll let you guys choose to merge it or not. Works on my machine, atleast.

Rexogamer commented on debia.html:
```diff
@@ -0,0 +1,46 @@
+        </header>
+        <h1>Debia</h1>
+        <slogan>The multi-purpose bot you've always wanted... rewritten.</slogan>
+        <p>Debia is a complete rewrite of Cytrus-RE, but it's not made in JS: It's made in TypeScript.</p>
```
```suggestion
        <p>Debia is a complete rewrite of Cytrus-RE, but it's not made in JS, it's made in TypeScript.</p>
```

Oxygemo commented:
By the way, if you want descs, you can do like me:

git commit -s -m "commit title" -m "commit desc"

You don't need the -s, but I use it because it's, well, formal.

Rexogamer commented on dev/commandblock6417.html:
```diff
@@ -1,40 +1,10 @@
-        <a href="https://github.com/Commandblock6417">GitHub Profile</a>
-        <a href="https://steamcommunity.com/id/0017534">Steam profile</a>
-        <a href="https://twitter.com/Commandblock641">Twitter profile</a>
+        <h1> Seems like you have stumbled upon an obsolete site, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Devnol.html">here to get to the new site.</a></h1>
```
```suggestion
        <h1> Seems like you have stumbled upon an obsolete page, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Devnol.html">here to get to the new site.</a></h1>
```

Rexogamer commented on dev/odyssey346.html:
```diff
@@ -1,36 +1,10 @@
-        <a href="https://github.com/Oxygemo">GitHub Profile</a>
-        <a href="https://steamcommunity.com/id/Odyssey346">Steam profile</a>
-        <a href="https://twitter.com/odyssey346">Twitter profile</a>
+        <h1> Seems like you have stumbled upon an obsolete site, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Odyssey346.html">here to get to the new site.</a></h1>
```
```suggestion
        <h1> Seems like you have stumbled upon an obsolete page, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Odyssey346.html">here to get to the new site.</a></h1>
```

Rexogamer commented on meetthedevs.html:
```diff
@@ -18,14 +18,16 @@
             </div>
         </header>
-        <h1>In this webpage you can read about the developers behind the Terrific Tea Studios team.</h1>
+        <h1>This site is actually obsolete, we've moved these to TTS' page instead (to not have to update both sites at once). Click any button to be sent to the (actual) bios!</h1>
```
```suggestion
        <h1>This part of the site is actually obsolete as we've moved these to TTS' site instead (to not have to update both sites at once). Click any button to be sent to the (actual) bios!</h1>
```

Rexogamer commented on meetthedevs.html:
```diff
@@ -18,14 +18,16 @@
         </header>
-        <h1>In this webpage you can read about the developers behind the Terrific Tea Studios team.</h1>
+        <h1>This site is actually obsolete, we've moved these to TTS' page instead (to not have to update both sites at once). Click any button to be sent to the (actual) bios!</h1>
+        <p>Or alternatively just go to the new bio page where there's more members and it's cleanier.</p>
```
```suggestion
        <p>Or alternatively just go to the new bio page where there's more members and it's cleaner.</p>
```

Rexogamer reviewed (changes requested):
Almost there :)

Oxygemo commented:
these are great suggestions, thanks for noticing! I'll get right to work.

Oxygemo commented:
Done. Commiting changes right now

Commit:
Do rexo's suggestions

Thanks for noticing!

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Oxygemo commented:
Testing my email signature

----------------------------------
Alexander/Odyssey346
-----------------------------------

Oxygemo commented:
Trying again

On Fri, Dec 11, 2020 at 8:06 PM Odyssey346 <alej0hio2007@gmail.com> wrote:

>
> Testing my email signature
>
> ----------------------------------
> Alexander/Odyssey346
> -----------------------------------
>


-- 
Alexander/Odyssey346

Sorta dev @ https://terrific-tea.github.io

Commit:
I forgot one thing

That was for testing purposes and I forgot to deelt it. Sorry I did a fucky wucky uwu!!!!

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

Oxygemo reviewed (commented):
lgtm go merge pls

Commit:
Formatting on the redirect sites

penis cock :DDDDDDD

Signed-off-by: Oxygemo <alej0hio2007@gmail.com>
